corehq/ex-submodules/dimagi/utils/post.py

```diff
@@ -33,30 +33,33 @@ def simple_post(data, url, content_type="text/xml", timeout=60, headers=None, au
     if auth:
         kwargs["auth"] = auth
 
+    if verify:
```

Comment:
I think you want this to be:
```py
if verify is not None:
``` 
otherwise, this won't pass the parameter on when `verify = False`.

Reply:
Right. updated.